target/linux/ar71xx/config-4.4

```diff
@@ -253,6 +253,7 @@ CONFIG_ATH79_MACH_WZR_HP_G300NH=y
 CONFIG_ATH79_MACH_WZR_HP_G300NH2=y
 CONFIG_ATH79_MACH_WZR_HP_G450H=y
 CONFIG_ATH79_MACH_XD3200=y
+CONFIG_ATH79_MACH_XD6700=y
```

Comment:
Please, add this line also in `target/linux/ar71xx/config-4.9`. Plus, add `# CONFIG_ATH79_MACH_XD6700 is not set` in `target/linux/ar71xx/nand/config-default` and `target/linux/ar71xx/mikrotik/config-default`.

Reply:
Have these files changed as I am getting a conflict ?

Reply:
> Have these files changed as I am getting a conflict ?

`git log` should give you an answer!